src/placeos-models/asset.cr

```diff
@@ -0,0 +1,64 @@
+require "./base/model"
+
+module PlaceOS::Model
+  class Asset < ModelBase
+    include RethinkORM::Timestamps
+
+    table :asset
+
+    attribute name : String, es_subfield: "keyword"
+    attribute category : String = ""
+    attribute description : String = ""
+
+    # # TO:DO field needs further defition
+    attribute size : String = ""
+
+    attribute purchase_date : Time
+    attribute good_until_date : Time?
+
+    attribute barcode : Int32
+    attribute brand : String = ""
+
+    # Saved as cents
+    attribute purchase_price : Int32
+
+    # Array of URLs to images for an asset
+    attribute images : Array(String) = [] of String
+
+    # URL of downloadable receipt
+    attribute invoice : String?
+
+    attribute quantity : Int32 = 0
+    attribute in_use : Int32 = 0
+
+    attribute other_data : JSON::Any = JSON::Any.new({} of String => JSON::Any), converter: JSON::Any::StringConverter, es_type: "object"
```

Comment:
Why `JSON::Any::StringConverter`? Can we store the JSON normally?

ATM, I believe this would result in something like `{"other_data":"\{\"an\"\:\"object\"\}"}`